drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -94,3 +96,10 @@ config ETH_STM32_CARRIER_CHECK_RX_IDLE_TIMEOUT_MS
 	  PHY's carrier status is re-evaluated.
 
 endif # ETH_STM32_HAL
+
+config NOCACHE_MEMORY
```

Comment:
Maybe this Kconfig setting should be *namespaced*?

Reply:
Can you give me some example for what you mean?

Reply:
Some sort of Kconfig option prefix to avoid possible future name clashes (e.g. `ETH_STM32_NOCACHE_MEMORY`). I'm not sure if there is an established policy. @erwango @carlescufi ?

Reply:
Good question. I would wait until any suggestion from both comes.

Reply:
`ETH_STM32_NOCACHE_MEMORY` would do, thanks @gmarull for the catch.

Reply:
Changed as suggested.